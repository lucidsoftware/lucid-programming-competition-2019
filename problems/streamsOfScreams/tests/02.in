50
lorem
ipsum
dolor
sit
amet,
consectetur
adipiscing
elit.
aliquam
aliquam
nisi
vitae
ullamcorper
vestibulum.
ut
finibus
elit
sed
dolor
vestibulum,
ghost
aliquam
nisl
pretium.
nulla
boo!
dolor
ghost
efficitur
mattis.
morbi
id
sapien
mattis,
ultrices
velit
eleifend,
blandit
turpis.
nulla
pharetra
tempor
augue,
eu
egestas
arcu
ultrices
id.
phasellus
ultrices.