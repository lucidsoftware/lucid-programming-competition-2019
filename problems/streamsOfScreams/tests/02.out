Ah!
Ahhh!!!
Ah!
5